### DISCUSSION TITLE
Revert back to original firmware

### DISCUSSION THREAD

**Discussion by tavicu** [themactep/thingino-firmware#310] [General]
Hei.

I had multiple cameras from Wyze (v3) that stopped working, not finding WiFi network. I have installed thingino on them and they are all working except one. The wifi network appears but when I try to connect to it and setup it fails to connect.

Is there any option to revert back to original firmware so I can use it as a webcam with wz_mini?

Or is there something more I could look into logs or something, maybe I can find out why I can't connect to it?

Thank you

**Comment by gtxaspec** [comment]
can you run `thingino-diag -l` on the device (via UART serial) and share? We can try to debug.  Also, make sure the antenna is securely connected inside the device.
